src/components/Identityprovider/identityprovider.component.js

```diff
@@ -74,11 +74,12 @@ export async function authenticate(req, res, next) { // eslint-disable-line comp
       return;
     }
 
+    const stickyLibrary = req.cookies ? req.cookies.stickyLibrary : null;
```

Comment:
kan stickyLibrary ikke risikere at være undefined eller lign? Vil der være potentiale for noget grimt med det?